src/dmd/inline.d

```diff
@@ -123,69 +123,59 @@ public:
     override void visit(CompoundStatement s)
     {
         //printf("CompoundStatement.doInlineAs!%s() %d\n", Result.stringof.ptr, s.statements.dim);
+        visitCompoundStatement(s, 0);
+    }
+
+    private void visitCompoundStatement(CompoundStatement s, size_t n)
```

Comment:
Document what `n` is.

Reply:
This would appear to be better served as a function nested within `visit(CompoundStatement)`.

Reply:
And perhaps give it a more clear.

Reply:
@WalterBright - I don't think so.  `visitCompoundStatement` can be called from a delegate passing a different `this` context, see `visitIfStatement`.

Reply:
@jacob-carlborg - A more clear what?  Name?  It's just a counter var... startFrom?

Reply:
Ok, but that means even more that `n` needs to be documented. Might also consider making it an actual slice!

Reply:
> A more clear what? Name?

@ibuclaw yes, a more clear name.

> It's just a counter var... startFrom?

I don't know what it's counting. I don't like having to read the whole function to figure out what the parameters are used for. If the parameters have descriptive names it a lot easier to figure out what they're used for just by looking at the signature.